x/subspaces/keeper/subspaces_test.go

```diff
@@ -346,6 +360,12 @@ func (suite *KeeperTestsuite) TestKeeper_DeleteSubspace() {
 				// Make sure the permissions are deleted
 				permissions := suite.k.GetSubspaceUserPermissions(ctx, 1)
 				suite.Require().Empty(permissions)
+
+				// Make sure the user grants are deleted
+				suite.Require().Empty(suite.k.GetSubspaceUserGrants(ctx, 1))
```

Comment:
I prefer splitting the declaration and check. Even though more verbose, it makes it easier to read
```suggestion
				userGrants := suite.k.GetSubspaceUserGrants(ctx, 1)
				suite.Require().Empty(userGrants)
```